src/ForgotPasswordController.js

```diff
@@ -52,7 +54,7 @@ function (Okta, FormController, Enums, FormType, ValidationUtil, Util, ContactSu
       }
     },
     getTemplateData: function () {
-      return this.settings.pick('helpSupportNumber');
+      return this.settings.pick('helpSupportNumber', 'features');
```

Comment:
probably only `pick` `features.hideBackToSignInForReset`

Reply:
Doesn't work for deep picking object.